ledger-api/grpc-definitions/com/daml/ledger/api/v1/active_contracts_service.proto

```diff
@@ -38,23 +38,26 @@ message GetActiveContractsRequest {
   // In particular, setting the verbose flag to true triggers the ledger to include labels for record fields.
   // Optional
   bool verbose = 3;
+
+  // The offset at which the snapshot of the active contracts will be computed.
+  // Must be no greater than the current ledger end offset.
```

Comment:
I assume it has to be greater than the latest pruning offset? Otherwise it seems like you'll omit contracts that have been active at the specified offset but have been archived before the pruning interval.

Reply:
Current implementation allows active_at offset that is greater than or equal to the pruning offset.

Added conformance tests verifying that and a note in the proto file